fuel-core/src/database/metadata.rs

```diff
@@ -1,24 +1,41 @@
+use crate::config::Config;
 use crate::database::columns::METADATA;
 use crate::database::Database;
 use crate::model::BlockHeight;
 use crate::state::Error;
 
+pub(crate) const DB_VERSION_KEY: &[u8] = b"version";
 pub(crate) const CHAIN_NAME_KEY: &[u8] = b"chain_name";
 pub(crate) const CHAIN_HEIGHT_KEY: &[u8] = b"chain_height";
 pub(crate) const FINALIZED_DA_HEIGHT_KEY: &[u8] = b"finalized_da_height";
 pub(crate) const VALIDATORS_DA_HEIGHT_KEY: &[u8] = b"current_validator_set";
 pub(crate) const LAST_COMMITED_FINALIZED_BLOCK_HEIGHT_KEY: &[u8] =
     b"last_commited_finalized_block_height";
 
+pub(crate) const DB_VERSION: u32 = 0;
```

Comment:
We should cleanup existing db version logic as well:

https://github.com/FuelLabs/fuel-core/blob/master/fuel-core/src/database.rs#L55
https://github.com/FuelLabs/fuel-core/blob/master/fuel-core/src/state/rocks_db.rs#L57

Reply:
I didnt even see that there is a version there, I moved constants in `metadata.rs`